src/omotes_simulator_core/adapter/transforms/esdl_asset_mapper.py

```diff
@@ -94,6 +95,7 @@ def to_entity(self, esdl_asset: EsdlAssetObject) -> ControllerProducer:
             temperature_supply=temperature_supply,
             temperature_return=temperature_return,
             power=power,
+            marginal_costs=marginal_costs,
```

Comment:
Do you want to use the marginal cost in the ControllerProducer? I would suggest to switch to a variable name that has a clear definition about how it is being used.

Reply:
I would keep it marginal costs, later we might create a controller which works on another input. This also makes it clear for us what it is and where it comes from. In the controller I have a method which sets priority based on marginal costs.
So then it is logical to have an attribute marginal costs.

Reply:
Can you make a comment in the docstring method so that it is clear that we use the 'marginal_cost' as a proxy for priority?

Reply:
Added it in the rst documentation